plugins/notifications/api-report.md

```diff
@@ -103,6 +103,7 @@ export const NotificationsSidebarItem: (props?: {
   titleCounterEnabled?: boolean;
   snackbarEnabled?: boolean;
   snackbarAutoHideDuration?: number | null;
+  markAsReadOnLinkOpen?: boolean;
```

Comment:
I wonder if there's really much variance here across apps? I.e. either it should be the built-in behavior, or it might have some problems that we should solve regardless. I'm thinking we should aim to not need to have this configurable

Reply:
I would vote for making this the default behavior but not sure what for example @mareklibra thinks?

Reply:
I don't think we need this to be configurable.